packages/@tinacms/toolkit/src/packages/react-sidebar/components/Nav.tsx

```diff
@@ -26,7 +26,8 @@ import { FormModal } from '../../react-forms'
 import { useEditState } from '@tinacms/sharedctx'
 import type { ScreenPlugin } from '../../react-screens'
 import { LoadingDots } from '../../form-builder'
-import { SyncStatus } from './SyncStatus'
+import { SyncStatus, SyncErrorWidget } from './SyncStatus'
+import { useCMS } from '../../react-core'
```

Comment:
Our monorepo setup seems to be handling this (I see our other imports do the same thing). But to confirm, is importing like this from another sibling package okay? @logan-anderson @jeffsee55

Reply:
@jamespohalloran yes because these aren't really packages, they're just folders from when we moved all of this to be part of `@tinacms/toolkit`.